tools/notifier/term.py

```diff
@@ -14,6 +14,7 @@
 # limitations under the License.
 
 from __future__ import print_function, division, absolute_import
+from past.builtins import basestring
```

Comment:
Doesn't look like this import is used anywhere

Reply:
It's not intuitive at all, but including basestring in this way affects how string concatenation occurs in line 77.

Reply:
Ah I see, thanks for clarifying. I just checked the cheatsheet and I see what you're talking about: http://python-future.org/compatible_idioms.html#basestring

If you end up pushing more commits to this, do you think you could add a comment pointing that reference so others know why its there?

Reply:
Ah, yeah. Will do. There's been so much back and forth between testing across two OSs that I let commit quality suffer.

Reply:
Oh no its no big deal, its honestly fine how it is. Just trying to make it easier for future maintenance.